testutil/apiload/main.go

```diff
@@ -124,25 +128,26 @@ func main() {
 		vegAttacker := vegeta.NewAttacker(vegeta.MaxBody(100))
 
 		wg.Add(1)
-		go func(attacker *Attacker) {
+		go func(attacker *Attacker, id int) {
```

Comment:
Code change: Do not capture loop variable in closure, especially since it is run async.